Source/Fuse.Triggers/Actions/Visibility.uno

```diff
@@ -31,7 +31,11 @@ namespace Fuse.Triggers.Actions
 	{
 		protected override void Perform(Node target)
 		{
-			if (target is IShow) ((IShow)target).Show();
+			var t = target.FindByType<IShow>();
```

Comment:
In most cases it'll find the node just 2 parent's away. For most actions we actually wish to defer the scanning for perofrmance reasons. It'd be bad if all actions resolved themselves during the rooting phase -- it'd take a lot longer.  Of course, it'd be nicer for detecting errors statically. :/